guac-install.sh

```diff
@@ -225,44 +233,59 @@ elif [[ "${NAME}" == *"Debian"* ]] || [[ "${NAME}" == *"Raspbian GNU/Linux"* ]]
     else
         LIBPNG="libpng12-dev"
     fi
-    if [ "$installMySQL" = true ]; then
+    if [ "${installMySQL}" = true ]; then
         MYSQL="default-mysql-server default-mysql-client mysql-common"
     # Checking if (any kind of) mysql-client or compatible command installed. This is useful for existing mariadb server
-    elif [ -x "$(command -v mysql)" ]; then
+    elif [ -x "$( command -v mysql )" ]; then
         MYSQL=""
     else
         MYSQL="default-mysql-client"
     fi
 else
-    echo "Unsupported Distro - Ubuntu, Debian, Kali or Raspbian Only"
+    echo "Unsupported distribution - Debian, Kali, Raspbian or Ubuntu only"
     exit 1
 fi
 
-# Update apt so we can search apt-cache for newest tomcat version supported & libmysql-java
+# Update apt so we can search apt-cache for newest Tomcat version supported & libmariadb-java/libmysql-java
 echo -e "${BLUE}Updating apt...${NC}"
 apt-get -qq update
 
-# Check if libmysql-java is available
-if [[ $(apt-cache show libmysql-java 2> /dev/null | egrep "Version:" | wc -l) -gt 0 ]]; then
+# Check if libmariadb-java/libmysql-java is available
+# Debian 10 >= ~ https://packages.debian.org/search?keywords=libmariadb-java
+if [[ $( apt-cache show libmariadb-java 2> /dev/null | egrep "Version:" | wc -l ) -gt 0 ]]; then
+    # When something higher than 1.1.0 is out ~ https://issues.apache.org/jira/browse/GUACAMOLE-852
+    #echo -e "${BLUE}Found libmariadb-java package...${NC}"
+    #LIBJAVA="libmariadb-java"
+    # For v1.1.0 and lower
+    echo -e "${YELLOW}Found libmariadb-java package (known issues). Will download libmysql-java ${MCJVER} and install manually${NC}"
+    LIBJAVA=""
+# Debian 9 <= ~ https://packages.debian.org/search?keywords=libmysql-java
+elif [[ $( apt-cache show libmysql-java 2> /dev/null | egrep "Version:" | wc -l ) -gt 0 ]]; then
+    echo -e "${BLUE}Found libmysql-java package...${NC}"
     LIBJAVA="libmysql-java"
 else
+    echo -e "${YELLOW}lib{mariadb,mysql}-java not available. Will download mysql-connector-java-${MCJVER}.tar.gz and install manually${NC}"
     LIBJAVA=""
-    echo -e "${YELLOW}libmysql-java not available. Will download ${MCJVER} and install manually${NC}"
 fi
-echo
 
 # tomcat9 is the latest version
 # tomcat8.0 is end of life, but tomcat8.5 is current
 # fallback is tomcat7
-if [[ $(apt-cache show tomcat9 2> /dev/null | egrep "Version: 9" | wc -l) -gt 0 ]]; then
+if [[ $( apt-cache show tomcat9 2> /dev/null | egrep "Version: 9" | wc -l ) -gt 0 ]]; then
+    echo -e "${BLUE}Found tomcat9 package...${NC}"
     TOMCAT="tomcat9"
-elif [[ $(apt-cache show tomcat8 2> /dev/null | egrep "Version: 8.[5-9]" | wc -l) -gt 0 ]]; then
+elif [[ $( apt-cache show tomcat8 2> /dev/null | egrep "Version: 8.[5-9]" | wc -l ) -gt 0 ]]; then
+    echo -e "${BLUE}Found tomcat8 package...${NC}"
+    TOMCAT="tomcat8"
+elif [[ $( apt-cache show tomcat7 2> /dev/null | egrep "Version: 8.[5-9]" | wc -l ) -gt 0 ]]; then
```

Comment:
This is still looking for Tomcat7 but grepping for version 8?

Reply:
Correct - re-used the code without altering.

Will address

Good catch